vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -225,6 +230,8 @@ def check_parse_tree(self, root, scope):
                     self.process_is_assignment(node, scope)
                 elif type(node.get_stmt()) == Assignment:
                     self.process_maybein(node, scope)
+                elif type(node.get_stmt()) == Event:
+                    self.process_event(node, scope)
 
     def check_parse_tree_function(self, function, scope):

```

Comment:
If you're not going to use typing in the function definition, you should have documentation in the function header that specifies the type of the arguments.